examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -208,3 +211,62 @@
         fontsize=20
     )
 plt.show()
+
+##############################################################################
+# 4. Learn Then Test
+# ------------------
+# In this part, we will use learn then test to control precision.
+# At the opposite of the 2 previous method, LTT can handle non-monotonous loss.
+# The procedure consist in multiple hypothesis testing. This is why the output
+# of this procedure isn't reduce to one value of :math: `\lambda`.
+
+# More precisely, we look after all the :math: `\lambda` that sastisfy the
+# following:
+# :math: `\mathbb{P}(R(\mathcal{T}_{\lambda}) \leq \alpha ) \geq 1 - \delta`,
+# where :math: `R(\mathcal{T}_{\lambda})` is the risk we want to control and
+# each $\lambda$ should satisfy FWER control. :math:`\alpha`
+# is the desired risk.
+
+# Notice that the procedure will diligently examine each :math: `\lambda`
+# such that the risk remains below level :math: `\alpha`, meaning not
+# every : math:`\lambda` will be considered.
+# This means that a for a :math:`\lambda` such that risk is below
+# :math:`\alpha`
+# doesn't necessarly pass the FWER control! This is what we are going to
+# explore.
```

Comment:
```suggestion
# The procedure consist in multiple hypothesis testing. This is why the output
# of this procedure isn't reduce to one value of :math:`\lambda`.
#
# More precisely, we look after all the :math:`\lambda` that sastisfy the
# following:
# :math:`\mathbb{P}(R(\mathcal{T}_{\lambda}) \leq \alpha ) \geq 1 - \delta`,
# where :math:`R(\mathcal{T}_{\lambda})` is the risk we want to control and
# each :math:`\lambda` should satisfy FWER control. :math:`\alpha`
# is the desired risk.
#
# Notice that the procedure will diligently examine each :math:`\lambda`
# such that the risk remains below level :math:`\alpha`, meaning not
# every :math:`\lambda` will be considered.
# This means that a for a :math:`\lambda` such that risk is below
# :math:`\alpha`
# doesn't necessarly pass the FWER control! This is what we are going to
# explore.
```

Reply:
@SZiane, Can you check that these typos are also present in the notebook? If so, can you correct them?

Reply:
Done